examples/sklearnex/model_builders_xgboost.py

```diff
@@ -0,0 +1,85 @@
+#===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+# daal4py Gradient Bossting Classification model creation from XGBoost example
+
+import daal4py as d4p
+import xgboost as xgb
+import numpy as np
+import pandas as pd
+
+
+def pd_read_csv(f, c=None, t=np.float64):
+    return pd.read_csv(f, usecols=c, delimiter=',', header=None, dtype=t)
+
+
+def main(readcsv=pd_read_csv, method='defaultDense'):
```

Comment:
Method is not used. Same for second example.
```suggestion
def main(readcsv=pd_read_csv):
```

Reply:
> Method is not used. Same for second example.

done